lib/db.js

```diff
@@ -1,35 +1,51 @@
-var logger = require('juttle/lib/logger').getLogger('opentsdb-init');
-var _ = require('underscore');
-var opentsdb = require( 'opentsdb');
-var OpentsdbSocket = require('opentsdb-socket');
-var Promise = require('bluebird');
-var client = Promise.promisifyAll(opentsdb.client());
-var socket = Promise.promisifyAll(OpentsdbSocket());
-
-var DB = {
-    init: function(config) {
-        if (config.host) {
-            client.host(config.host);
+'use strict';
+
+/* global JuttleAdapterAPI */
+let logger = JuttleAdapterAPI.getLogger('opentsdb-init');
+
+let _ = require('underscore');
+let opentsdb = require( 'opentsdb');
+let OpentsdbSocket = require('opentsdb-socket');
+let Promise = require('bluebird');
+let client = Promise.promisifyAll(opentsdb.client());
```

Comment:
Is this single global client safe? Like if I start of one paged/live read from 1 id and then in the middle of that kick off another one from another id won't the first one start reading from the second id?